### ISSUE TITLE
Place new topics from a pinned-spinner at the centre of the user's current view, not the (0,0) coordinate.

### ISSUE THREAD

**Issue by Bortseb** [metamaps/metamaps#801]
Depending on where the user pins the spinner currently... they might have no idea their new topics are successfully being created... because they are being placed somewhere entirely out of their view...

Also, with this enhancement... the user could intentionally change their view to intentional start a new spiral... which seems simple, but a significant improvement to me.

is this a simple enhancement @Connoropolous ? or I am missing something based on not knowing how things work?

**Comment by Connoropolous** [comment]
just a process of making it more flexible is all. not that hard, but not tiny either

**Comment by devvmh** [comment]
@bortseb this is related to my last comment on #739

**Comment by Bortseb** [comment]
Sort of @devvmh , given that I think it would be really useful to be able to decide at any time to start moving the group of topics that you've added "in bulk" (either from a pinned spinner or copy paste)

But I didn't realize that you were having issues actually moving your selected topics after pasting on the same map (since they are all overlapping)... thats unfortunate... maybe its just because the new topics aren't actually "on top"?

This issue is slightly different in that each newly created topic from a pinned spinner would need to get appended onto the selected set of topics.

**Comment by devvmh** [comment]
Yeah. The other relationship is about managing offsets/scale when managing topics

**Comment by Bortseb** [comment]
Not sure what you meant by this last point.

**Comment by devvmh** [comment]
I mentioned it in the other pr. We need to get good at understanding topic positioning relative to (0,0), the users view, and to each other. This is useful for cooy padte because of how it makes pasting into other maps more natural
